### PR TITLE
Add Dun & Bradstreet Deutschland GmbH

### PR DESCRIPTION
**Pull request by rugk**
AZDirect informed me that they got the address from this data broker („Abt. Customer Service” – if you want to add this for a possible letter), so obviously following the data trail would be great here…

As it si clearly a data broker, you may even consider adding it to the list of suggested companies.

### CODE REVIEW

File: companies/dud-de.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "phone": "+49 0615 13800",
+    "fax": "+49 0615 1380360",
+    "email": "info.de@dnb.com",
+    "web": "https://www-prod.dnb.com/de-de/",
+    "sources": [
+        "https://www-prod.dnb.com/de-de/impressum/"
+    ],
+    "request-language": "de",
+    "quality": "verified",
+    "nsfw": false
```
[the remark this answers is not in the document]
Comment (rugk) [reply companies/dud-de.json:21]: Well… your online tool allowed to select "false"… :upside_down_face:  

```suggestion
```

File: companies/dud-de.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "address": "Robert-Bosch-Straße 11\n64293 Darmstadt\nDeutschland",
+    "phone": "+49 0615 13800",
+    "fax": "+49 0615 1380360",
+    "email": "info.de@dnb.com",
+    "web": "https://www-prod.dnb.com/de-de/",
+    "sources": [
+        "https://www-prod.dnb.com/de-de/impressum/"
+    ],
+    "request-language": "de",
+    "quality": "verified",
```
Comment (rugk) [inline companies/dud-de.json:20]:
```suggestion
    "quality": "verified"
```

Is the comma makeing the bot cause parse errors?

**Review by mal-tee — APPROVED** [review]
Thank you!

File: companies/dud-de.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (mal-tee) [reply companies/dud-de.json]: Yeah, we aren't happy with the support for bools as well :/

**Comment by baltpeter** [comment]
It's quite hidden but they do have a [privacy policy](https://hello.dnb.com/rs/145-JUC-481/images/2022-02-21_Datenschutzhinweise-Auskunftsdatenbestand_DE.pdf) with a proper privacy email address, as well as [web forms](https://www-prod.dnb.com/de-de/datenschutz/ihre-rechte-als-betroffener/selbstauskunft-widerspruch/) for requests, which we can use to infer the required elements.

I'll update the record accordingly. And I'll also send a request myself. Would be great if a couple of other people from Germany would as well. If we can confirm that they have the data of multiple people, I'm happy to add them to the suggested companies.

**Comment by mal-tee** [comment]
Oh, thanks for catching that.

I already sent them a request via email. :)

**Comment by mal-tee** [comment]
They say that they don't have anything about me.

**Comment by baltpeter** [comment]
Same for me.

**Pull request merged**